src/util/token.js

```diff
@@ -3,17 +3,9 @@ const crypto = require('crypto') // sodium doesn't have HMAC
 const base64url = require('base64url')
 
 const TOKEN_HMAC_INPUT = 'token'
+const AUTH_HMAC_INPUT = 'authorization'
```

Comment:
As of right now, the auth token is the only secret value being used by plugin virtual. In previous versions though, the token was the only value being generated by the shared secret, and could have been done with the shared secret directly. Not using the shared secret directly lets us more with it in future versions.